### ISSUE TITLE
Hawtio:2.2.0 - Discover connections to local processes (Jolokia jvm agent 1.6.0)

### ISSUE THREAD

**Issue by ChromeTemplar** [hawtio/hawtio#2552]
I'm using hawtio 2.2.0 at my company.  I noticed that in order for Hawtio to discover the local processes, a local process has to have the jolokia jvm agent attached.

So when I have a local process with the jolokia jvm agent attached, Hawtio can discover the connection.  Cool!  

Then I see this:

![image](https://user-images.githubusercontent.com/14863288/50114842-a1945d80-0213-11e9-8880-e614fea055e5.png)

Small digression:  Note that clicking on the link for the agent opens a new browser (Chrome) tab that tries to connect.  The error message is that "This site can't be reached".  I do not think it makes sense to display this link if it can't be used, but 

On to the main point:  It is not obvious that clicking on the green circle does anything.  The mouse cursor does not change, and the tooltip does not show up.  By "green circle", I mean the "i.fa-4x.pficon-running" image.

But when I do, the browser (Chrome) opens a new tab, with a 404 error like this:

_This  user-vm1 page can’t be found 
No webpage was found for the web address: https://user-vm1:30501/?con=discover-10.1.54.92-43416-380d4bed-jvm_

If I then click on the "Remote" tab, the connection shows up:

![image](https://user-images.githubusercontent.com/14863288/50115024-372fed00-0214-11e9-8178-ee2bfed9d896.png)

When I click on the "Connect" button, Hawtio does connect to the process, so this part works as one would expect. (By the way, exploring the process this way is very hawt.)  

To sum up, I see two or three items that need to be fixed:

1. It is not obvious that the user needs to click on the green circle to create the connection.  Can the icon be changed to that the mouse cursor changes when the mouse cursor is hovering over the icon?  If not, can it be made more clear that clicking on the green circle is required?

2. Clicking on the green circle icon causes a new browser tab to open with an error, but the connection information is captured by Hawtio.  It would be better is the connection infomation

3.  Agent URL displayed even though the local host refuses to connect.

**Comment by alexkieling** [comment]
The feature to connect to local processes hasn't been improved since v1. I suggest using the feature to connect to remote processes, even if they are running locally.
